### PR TITLE
Embed gif

### PR DESCRIPTION
**Pull request by plsdev89**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Type: Message. Closes #000.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: packages/ckeditor5-media-embed/src/mediaembedediting.js [hunk 1]
```diff
@@ -7,6 +7,10 @@
  * @module media-embed/mediaembedediting
  */
 
+/* eslint-disable no-undef */
+/* eslint-disable no-useless-escape */
+/* eslint-disable max-len */
+
```
Comment (plsdev89) [inline packages/ckeditor5-media-embed/src/mediaembedediting.js:13]: These changes are for fixing es-lint errors on fjVideo regex.

File: packages/ckeditor5-media-embed/src/mediaembedediting.js [hunk 2]
```diff
@@ -209,7 +226,7 @@ export default class MediaEmbedEditing extends Plugin {
 					name: 'fjVideo',
 					url: [
 						/^https:\/\/fj-file-uploads\.s3\.us-east-2\.amazonaws\.com\/((?:[\w+]+(?:%[0-9A-Fa-f]{2}|\/))?fjvideo-[\w-]+).([\w-]+)/,
-  					/^https:\/\/cdn.flockjay.com\/((?:[\w+]+(?:%[0-9A-Fa-f]{2}|\/))?fjvideo-[\w-]+).([\w-]+)/,
+						/^https:\/\/cdn.flockjay.com\/((?:[\w+]+(?:%[0-9A-Fa-f]{2}|\/))?fjvideo-[\w-]+).([\w-]+)/,
```
Comment (plsdev89) [inline packages/ckeditor5-media-embed/src/mediaembedediting.js:231]: Just fixing eslint error

File: packages/ckeditor5-media-embed/src/mediaembedediting.js [hunk 3]
```diff
@@ -76,6 +80,19 @@ export default class MediaEmbedEditing extends Plugin {
...
+				{
+					name: 'gif',
+					url: [
+						/(http(s?):)([/|.|\w|\s|-])*\.gif/
+					],
+					html: match => {
+						const url = match[ 0 ];
+
+						return (
+							`<img src=${ url } style="width: 100%;" >`
```
Comment (suvan92) [inline packages/ckeditor5-media-embed/src/mediaembedediting.js]: @plsdev89 can we add an alt as well? Maybe capture the file name from the regex?

**Review by suvan92 — CHANGES_REQUESTED** [review]

File: packages/ckeditor5-media-embed/src/mediaembedediting.js (hunk 3 above)
Comment (plsdev89) [reply packages/ckeditor5-media-embed/src/mediaembedediting.js]: Done

**Review by suvan92 — APPROVED** [review]

**Pull request merged**